packages/tracking/package.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "@workleap/mixpanel",
+    "version": "1.0.0",
+    "license": "Apache-2.0",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/workleap/wl-tracking.git",
+        "directory": "packages/tracking"
+    },
+    "publishConfig": {
+        "access": "public",
+        "provenance": true
+    },
+    "type": "module",
+    "main": "./dist/index.js",
+    "exports": {
+        ".": {
+            "import": "./dist/index.js",
+            "types": "./dist/index.d.ts",
+            "default": "./dist/index.js"
+        }
+    },
+    "files": [
+        "dist"
+    ],
+    "scripts": {
+        "build": "rslib build --config ./rslib.build.ts",
+        "eslint": "eslint . --max-warnings=-0 --cache --cache-location node_modules/.cache/eslint",
+        "typecheck": "tsc",
+        "test": "vitest --config vitest.config.ts --no-watch"
+    },
+    "devDependencies": {
+        "@rsbuild/core": "1.2.19",
+        "@rslib/core": "0.5.5",
+        "@types/node": "22.13.11",
+        "@workleap/eslint-plugin": "3.3.1",
+        "@workleap/rslib-configs": "1.0.3",
+        "@workleap/typescript-configs": "3.0.2",
+        "eslint": "8.57.1",
+        "jsdom": "26.0.0",
```

Comment:
I recommend using "happy-dom" instead, this is the virtual DOM i've been using for all our other libs.